specs/protocol/tx_validity.md

```diff
@@ -106,12 +106,10 @@ def unavailable_balance(tx, col) -> int:
     """
     sentBalance = sum_outputs(tx, col)
     gasBalance = gasPrice * gasLimit
-    # Size excludes witness data as it is malleable (even by third parties!)
-    bytesBalance = size(tx) * bytePrice
     # Only native coin can be used to pay for gas
     if col != 0:
         return sentBalance
-    return sentBalance + gasBalance + bytesBalance
+    return sentBalance + gasBalance
```

Comment:
Similarly to my last post, I'd think we'd keep the byte balance costs included but just constrain to some fixed ratio of the gas price. Otherwise, we'd only meter txs based on execution and not capture the data cost anywhere.

Reply:
A little bit different from above, tx size usually falls under `gasLimit*gasPrice` as it is calculated as a gas it allows us to tweak it where gas is calculated. but I can revert that here and make a special rule for size as it was before.

Reply:
Before the addition of the bytePrice field, there was a constant in the specs for GAS_PER_BYTE and it was calculated here.

Reply:
Yeah we should keep the calculation here, just set rules on `bytesBalance` implicitly by containing `bytePrice`.